gnuboardProject/gnuboardProject/frontend/src/components/Portfolio/PortfolioWrite.tsx

```
@@ -286,6 +413,26 @@ const PortfolioWrite: React.FC = () => {
+      {/* 템플릿 미리보기 섹션 */}
+      <TemplateContainer>
+        <h2 style={{ marginBottom: '1rem' }}>템플릿 미리보기</h2>
+        <TemplateSelector
+          value={selectedTemplate}
+          onChange={(e) => setSelectedTemplate(e.target.value)}
+        >
+          <option value="default">기본 템플릿</option>
+          <option value="dark">다크 템플릿</option>
+          <option value="gradient">그라데이션 템플릿</option>
+          <option value="minimal">미니멀 템플릿</option>
+          <option value="art">아트 템플릿</option>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)